addons/netfox/rollback/network-rollback.gd

```diff
@@ -42,6 +42,12 @@ var input_redundancy: int:
 	set(v):
 		push_error("Trying to set read-only variable input_redundancy")
 
+var enable_state_diffs: bool:
+	get:
+		return ProjectSettings.get_setting("netfox/rollback/enable_state_diffs", true)
+	set(v):
+		enable_state_diffs = v
```

Comment:
Is the setter needless? I am not familiar with Godot's setters/getters.
Also are project settings' values cached on runtime, or does it search project settings' values each time I get()?

Reply:
Is there any reason for this flag to be writable during runtime? I'd just go with a `push_error` message.

`ProjectSettings.get_setting` boils down to a `Node.get` call under the hood, so I'd say it's fine as it is. But if you want to be proactive about perf, feel free to introduce a variable with an underscore prefix that's initialized in `_ready` and use that.

Reply:
I made it `push_error` as you suggested. As for the second part of this, I opened an issue #304